deployFactory.js

```diff
@@ -2,7 +2,7 @@
 const config = require("./config");
 const thorify = require("thorify").thorify;
 const Web3 = require("web3");
-const ConnexDriver = require('@vechain/connex-driver');
+const ConnexDriver = require("@vechain/connex-driver");
 
 const web3 = thorify(new Web3(), config.testnetRpcUrl);
```

Comment:
Thoughts on making the deploy scripts configurable for testnet vs mainnet? Obviously we will still need to update the contracts and bytecode until we can use the chainId function, so it might not matter so much if we have to edit these scripts to target mainnet...

Reply:
Yes I wanted to do that but I was having trouble figuring out how to pass arguments to a node script natively via npm. 

But I figured it out now, so I'll add a flag to let the deployer specify whether they want to deploy this on the testnet or mainnet. Thanks!